ocaml/testsuite/tests/typing-layouts-missing-cmi/function_arg.ml

```diff
@@ -10,8 +10,6 @@ module = "function_b.ml"
 script = "rm -f function_a.cmi"
 ***** expect
 *)
-(* CR layouts v2.9: all error messages below here are unreviewed *)
```

Comment:
```
       But the layout of Function_a.t must be a sublayout of '_representable_layout_1, because
         it's used as a function argument.
```
I left the same comment elsewhere, but this might be more intelligible as `But the layout of Function_a.t must be representable, because...`

These cases read especially oddly, because `'_representable_layout_1` is only introduced in this phrase and isn't referred to elsewhere, so there's no benefit to be had in naming it.